NitroxClient/MonoBehaviours/Multiplayer.cs

```diff
@@ -226,7 +226,6 @@ private void SceneManager_sceneLoaded(Scene scene, LoadSceneMode loadMode)
                 // If we just disconnected from a multiplayer session, then we need to kill the connection here.
                 // Maybe a better place for this, but here works in a pinch.
                 JoinServerBackend.StopMultiplayerClient();
-                SceneCleaner.Open();
```

Comment:
@Jannify Do you have any insights of the existence of this line ? Can't remember if you worked on that part

Reply:
I don't remember that line. I guess it's a safety precaution to ensure we have a totally clean slate game object wise.